### DISCUSSION TITLE
convert2rhel migration error

### DISCUSSION THREAD

**Discussion by vishnupjk** [oamg/convert2rhel#825] [General]
I am trying to migrate from centos 7.9 to rhel 7.9 using convert2rhel utility [custom repo].

I have performed 'yum update' and installed the gpg keys and convert2rhel utility as well.

while migration, I am getting the error message "no packages containing kernel modules available in the enabled repositories"

kindly help to fix this issue.

**Comment by ChrisPea** [comment]
yum install kernel-modules
